docs/custom-redirects.md

```diff
@@ -1,6 +1,7 @@
 ## Custom Netlify Redirects
 
-You can define custom redirects in a `_redirects` file.
+You can define custom redirects in a `_redirects` file for apps using the Essential Next.js plugin.
+
 The precedence of these rules are:
 
 - `_redirects`
```

Comment:
Just checking on the line below whether this should still read "`next-on-netlify` redirects" or whether it needs to be updated.

Reply:
oooo yes you could update the line below to " `Essential Next.js` redirects " 💡